gateway-service-definitions/src/main/resources/services/livy/0.4.0/service.xml

```diff
@@ -24,7 +24,9 @@
     </metadata>
   <routes>
     <route path="/livy/**?**"/>
-    <route path="/livy"/>
+    <route path="/livy">
+        <rewrite apply="LIVYSERVER/livy/outbound/headers/ui" to="response.headers"/>
```

Comment:
I think you want this to apply to the route after this as well. `/livy/`

Reply:
Added change to apply to `/livy/` path.